### ISSUE TITLE
Multiple values support for JQL `withChangedProperty `

### ISSUE THREAD

**Issue by hungbang** [javers/javers#919]
I followed this document: https://javers.org/documentation/jql-examples/#property-filter 
to implement JQL in my services: 
...
```
if (auditCriteria.getToCommitDate() != null) {
            builder.to(auditCriteria.getToCommitDate());
        }

        builder.withChangedProperty("name");
        
        JqlQuery jqlQuery = builder
            .withNewObjectChanges()
            .build();
        return javers.findChanges(jqlQuery);
```

But I have 2 changed properties need to filter: `name`, `key`.
Then my question is how can I use `OR` sematic for `changed properties`. 
For example:
`builder.withChangedPropertyIn(Lists.newArrayList("name", "key"));`

I have put the question on Stackoverflow as well. 
https://stackoverflow.com/questions/59074564/javers-and-spring-boot-changed-property-filter-for-multi-properties
I am very appreciated JaVers team who created nice audit open source. Thank you so much JaVers Team.

**Comment by bartoszwalacik** [comment]
There is no such filter, you can add it to Javers and contribute a pull request. Btw please do not crosspost. It makes our job harder.

**Comment by hungbang** [comment]
Thank you @bartoszwalacik , I've noted.

**Comment by alasun** [comment]
Can I work on this?

**Comment by bartoszwalacik** [comment]
sure, @alasun

**Comment by bartoszwalacik** [comment]
released in 5.13.0
